### ISSUE TITLE
Maybe the didChangeAudioRoute listeners callback should get a name returned by getAudioRoutes

### ISSUE THREAD

**Issue by nagyszili** [react-native-webrtc/react-native-callkeep#533]
# Bug report

- [ ] I've checked the [example](https://github.com/react-native-webrtc/react-native-callkeep/tree/master/example) to reproduce the issue.

- Reproduced on:
 - [ ] Android
 - [x] iOS

## Description

Value returned by ```getAudioRoute()``` :
```
[ { name: 'iPhone Microphone', type: 'Phone' },
  { name: 'Speaker', type: 'Speaker' },
  { name: 'Venue', type: 'Bluetooth' },
  { name: 'HG00595', type: 'Bluetooth' } ]
```

Selecting an audio route on the  call screen triggers the ```didChangeAudioRoute``` listener but the value what I get 
in case selecting the Venue is ```{ output: 'BluetoothHFP', reason: 3 }```
in case selecting the HG00595 is ```{ output: 'BluetoothHFP', reason: 3 }```
in case selecting the iPhone is ```{ output: 'Receiver', reason: 3 }```
in case selecting the Speaker is ```{ output: 'Speaker', reason: 4 }```

The output string value shouldn't be the name of the selected audioRoute, just like how is returned by the getAudioRoute ?

![image](https://user-images.githubusercontent.com/41357953/155301697-db636c0e-1794-4b2e-9b60-350958716339.png)

## Versions
    - Callkeep: 4.3.2
    - React Native: 0.64
    - iOS: 15.2
    - Android:
    - Phone model: 6s Plus, 12

**Comment by rcidt** [comment]
Yes, you are correct. As it is now, it makes it impossible to support multiple BT device selection from the app UI.

**Comment by MeghaPawar2410** [comment]
Hi 
I am facing the same issue. Any solution found for the same?

**Comment by nagyszili** [comment]
Well I've created a [pull request](https://github.com/react-native-webrtc/react-native-callkeep/pull/541) back then to fix this issue but its still open

**Comment by tsminh** [comment]
any update on this ?

**Comment by MeghaPawar2410** [comment]
We were able to resolve it by iOS native code only

**Comment by Romick2005** [comment]
@nagyszili Are you able to switch between connected bluetooth devices? 
How do you change the default last-in bluetooth output selection rule? 
To simplify are you able to switch from  { name: 'Venue', type: 'Bluetooth' } to  { name: 'HG00595', type: 'Bluetooth' }  in your case?

**Comment by iim-norse** [comment]
when plan merge this ?
